### ISSUE TITLE
🐛 Tina CMS - crashing on Windows 

### ISSUE THREAD

**Issue by amankumarrr** [SSWConsulting/SSW.Website#483]
### Description

When we run Tina CMS on the local Windows machine, it crashes on saving the changes. However, on WSL, it works fine.

We have created an issue on Tina's repo: https://github.com/tinacms/tina.io/issues/1540

```[tasklist]
### Tasks
- [ ] https://github.com/SSWConsulting/SSW.Website-v3/issues/484

**Comment by amankumarrr** [comment]
This issue has been resolved by https://github.com/SSWConsulting/SSW.Website-v3/pull/530. However, we still need to have a discussion on whether we need to create a dev container or not

**Comment by bradystroud** [comment]
Fixed in #530
Dev container discussion should be on #484
